types/svg-baker-runtime/src/sprite.d.ts

```diff
@@ -0,0 +1,26 @@
+import SpriteSymbol from './symbol';
+import SpriteConfig from './sprite.config';
+
+export default class Sprite {
+    config: SpriteConfig;
+    symbols: SpriteSymbol[];
+
+    constructor(config: SpriteConfig);
+
+    /**
+     * Add new symbol. If symbol with the same id exists it will be replaced.
+     * @return `true` - symbol was added, `false` - replaced
+     */
+    add(symbol: SpriteSymbol): boolean;
+
+    /**
+     * Remove symbol & destroy it
+     * @return `true` - symbol was found & successfully destroyed, `false` - otherwise
+     */
+    remove: (id: string) => boolean;
+    find: (id: string) => SpriteSymbol | null;
+    has: (id: string) => boolean;
+    stringify: () => string;
+    toString: () => string;
+    destroy: () => void;
```

Comment:
```suggestion
```
this one is probably protected/private or at least non documented (like in other places)
https://github.com/JetBrains/svg-mixer/blob/68f4f77f780d64883e26719a882c9893888c7099/packages/svg-baker-runtime/src/sprite.js#L82
https://github.com/JetBrains/svg-mixer/blob/68f4f77f780d64883e26719a882c9893888c7099/packages/svg-baker-runtime/src/sprite.js#L82